mopidy/local/search.py

```diff
@@ -3,7 +3,18 @@
 from mopidy.models import SearchResult
 
 
-def find_exact(tracks, query=None, uris=None):
+def find_exact(tracks, query=None, limit=100, offset=0, uris=None):
+    """
+    Filter a list of tracks where ``field`` is ``values``.
+
+    :param list tracks: a list of :class:`~mopidy.models.Track`
+    :param dict query: one or more queries to search for
```

Comment:
Maybe change to "one or more field/value pairs to search for"? In my head, there's a single query with multiple key-value pairs.